pages/1.12/cli/command-reference/dcos-plugin/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: layout.pug
+navigationTitle:  dcos plugin
+title: dcos plugin
+menuWeight: 12
+excerpt: Installing and managing DC/OS CLI plugins
+
+enterprise: false
+---
+
+# Description
+Install and manage DC/OS CLI plugins. Unlike `dcos package`, this is only used to manage CLI plugins locally on your computer and will have no effect on the cluster.
```

Comment:
Rewrite "Unlike `dcos package`, this is only used to manage CLI plugins locally on your computer and will have no effect on the cluster." as "Unlike `dcos package`, the `dcos plugin` is only used to manage CLI plugins locally on your computer and will have no effect on the cluster.